services/ingestion/etl/agri/csv/test/unit/lib/__snapshots__/transform.spec.js.snap

```diff
@@ -236,12 +260,16 @@ Object {
   "project_id": undefined,
   "project_locations": Array [
     Object {
+      "address": undefined,
```

Comment:
Are those `undefined` expected?

Reply:
I don't know. I just updated the snapshots as usual.